liquibase-core/src/main/java/liquibase/command/core/InternalHistoryCommandStep.java

```diff
@@ -27,13 +32,17 @@ public class InternalHistoryCommandStep extends AbstractCommandStep {
         DATE_FORMAT_ARG = builder.argument("dateFormat", DateFormat.class)
                 .defaultValue(DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT), "Platform specific 'short' format")
                 .build();
+        FORMAT_ARG = builder.argument("format", HistoryFormat.class)
```

Comment:
Same comment as in `HistoryCommandStep` class.

Reply:
Fixed in f6ed6dc3d314e65c67bc1c9a66a3b585c4603463